deploy/olm-catalog/kfctl/0.1.0/kfctl.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "Machine Learning"
+    description: "Kubeflow Operator for deployment and management of Kubeflow"
+    support: Kubeflow
+  name: kfctl.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KfDef is the Schema for the applications API
+      kind: KfDef
+      name: kfdefs.kfdef.apps.kubeflow.org
+      version: v1
+      displayName: KubeFlow
+  description: "Kubeflow Operator for deployment and management of Kubeflow"
+  displayName: Kubeflow
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - nonResourceURLs:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: cluster-admin
```

Comment:
This needs to be `kubeflow-operator` after the code gen.

Reply:
done